### PR TITLE
feat(telecom): add beta softphone

### PR DESCRIPTION
**Pull request by antonymarion**
ref: MANAGER-13706

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat#MANAGER-13706
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
~~- [ ] Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -5,6 +5,10 @@
   "telephony_line_tab_consumption": "Consommation",
   "telephony_line_tab_call_management": "Gestion des appels",
   "telephony_line_tab_music_management": "Gestion des musiques",
+  "telephony_line_tab_softphone": "Softphone",
+  "telephony_line_tab_beta_softphone_description": "Paramétrez ici votre option Softphone. A noter que pendant notre phase Béta, si vous choisissez d'avtiver le Softphone sur votre ligne, nous ne pourrez plus la provisionner ailleurs que sur nos applications de Softphone, « Softcall » (et donc pas sur un terminal de téléphonie fixe ou sur un autre logiciel de Softphonie)",
+  "telephony_line_tab_beta_softphone_active_description": "Si l'interrupteur est actif, alors cette ligne est utilisable sur l'ensemble de vos applications « Softcall »: mobile et ordinateur",
+  "telephony_line_tab_beta_softphone_inactive_description": "Si l'interrupteur est gris, alors cette ligne est utilisable uniquement sur un terminal de téléphonie fixe (filaire ou DECT); ou sur un softphone de votre choix",
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json:11]:
```suggestion
  "telephony_line_tab_beta_softphone_inactive_description": "Si l'interrupteur est inactif, alors cette ligne est utilisable uniquement sur un terminal de téléphonie fixe (filaire ou DECT) ou sur un softphone de votre choix.",
```
To maintain logic in activation

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -5,6 +5,10 @@
   "telephony_line_tab_consumption": "Consommation",
   "telephony_line_tab_call_management": "Gestion des appels",
   "telephony_line_tab_music_management": "Gestion des musiques",
+  "telephony_line_tab_softphone": "Softphone",
+  "telephony_line_tab_beta_softphone_description": "Paramétrez ici votre option Softphone. A noter que pendant notre phase Béta, si vous choisissez d'avtiver le Softphone sur votre ligne, nous ne pourrez plus la provisionner ailleurs que sur nos applications de Softphone, « Softcall » (et donc pas sur un terminal de téléphonie fixe ou sur un autre logiciel de Softphonie)",
+  "telephony_line_tab_beta_softphone_active_description": "Si l'interrupteur est actif, alors cette ligne est utilisable sur l'ensemble de vos applications « Softcall »: mobile et ordinateur",
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json:10]:
```suggestion
  "telephony_line_tab_beta_softphone_active_description": "Si l'interrupteur est actif, alors cette ligne est utilisable sur l'ensemble de vos applications « Softcall » : mobile et ordinateur.",
```
I don't know if ":" should be replace by : mobile et ordinateur => (mobile et ordinateur)

**Review by ghyenne — CHANGES_REQUESTED** [review]

**Review by qpavy — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json (hunk 2 above)
Comment (antonymarion) [reply packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json:10]: idk either

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.routing.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      },
+      resolve: {
+        softphoneBetaElligibility: /* @ngInject */ (
+          SoftphoneService,
+          $stateParams,
+        ) =>
+          SoftphoneService.getSoftphoneBetaElligibility(
+            $stateParams.billingAccount,
+            $stateParams.serviceName,
+          ),
```
[the remark this answers is not in the document]
Comment (antonymarion) [reply packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.routing.js:19]: Right!

**Review by ghyenne — APPROVED** [review]

**Pull request merged**